.github/workflows/ci.yaml

```diff
@@ -39,63 +39,28 @@ jobs:
 
   test:
     runs-on: ubuntu-latest
-    # Define any services needed for the test suite (or delete this section)
-    # services:
-    #   postgres:
-    #     image: postgres:16
-    #     ports:
-    #       - "5432:5432"
-    #     env:
-    #       POSTGRES_PASSWORD: secretpassword
     env:
       BATON_LOG_LEVEL: debug
-      # Add any environment variables needed to run baton-snowflake
-      # BATON_BASE_URL: 'http://localhost:8080'
-      # BATON_ACCESS_TOKEN: 'secret_token'
-      # The following parameters are passed to grant/revoke commands
-      # Change these to the correct IDs for your test data
-      CONNECTOR_GRANT: 'grant:entitlement:group:1234:member:user:9876'
-      CONNECTOR_ENTITLEMENT: 'entitlement:group:1234:member'
-      CONNECTOR_PRINCIPAL: 'user:9876'
-      CONNECTOR_PRINCIPAL_TYPE: 'user'
     steps:
       - name: Install Go
         uses: actions/setup-go@v4
         with:
           go-version: 1.22.x
       - name: Checkout code
         uses: actions/checkout@v3
-      # Install any dependencies here (or delete this)
-      # - name: Install postgres client
-      #   run: sudo apt install postgresql-client
-      # Run any fixture setup here (or delete this)
-      # - name: Import sql into postgres
-      #   run: psql -h localhost --user postgres -f environment.sql
-      #   env:
-      #     PGPASSWORD: secretpassword
       - name: Build baton-snowflake
         run: go build ./cmd/baton-snowflake
-      # - name: Run baton-snowflake
-      #   run: ./baton-snowflake
-
+      - name: Run baton-snowflake
+        run: ./baton-snowflake
```

Comment:
I think you need env vars for BATON_ACCOUNT_IDENTIFIER, BATON_ACCOUNT_URL, BATON_PUBLIC_KEY_FINGERPRINT, and BATON_USER_IDENTIFIER. But I don't think we have a snowflake account to test against so. :(